uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -652,7 +652,11 @@
             -->
             <chunk-point/> <!-- Performance Optimization, see ChunkPointPlaceholderEventSource -->
         </head>
-        <body class="up dashboard portal fl-theme-mist">
+        <body>
+          <xsl:attribute name="class">
+            <xsl:text>up dashboard portal fl-theme-mist</xsl:text>
```

Comment:
I'm not sure that `fl-theme-mist` is needed any longer.  I'd be curious to know if anyone can perceive a difference without it.